src/txdb.cpp

```diff
@@ -452,6 +673,16 @@ bool CBlockTreeDB::ReadFlag(const std::string &name, bool &fValue) {
     return true;
 }
 
+bool CBlockTreeDB::blockOnchainActive(const uint256 &hash) {
```

Comment:
We might move this in the ifdef above; the same also in the header file.

Reply:
Fixed.